File: C# Fundamentals/11. Arrays/P01_DayOfWeek.cs
//int[] array = { 1, 2, 3, 4, 5, 100 };

//for (int i = 0; i < array.Length; i++)
//{
//    Console.WriteLine(array[i] + " ");
//}

string[] days =
{
    "Monday",
    "Tuesday",
    "Wednesday",
    "Thursday",
    "Friday",
    "Saturday",
    "Sunday"
};

int currentDay = int.Parse(Console.ReadLine());

if (currentDay >= 1 && currentDay <= 7) 
{
    Console.WriteLine(days[currentDay -1]);
}
else
{
    Console.WriteLine("Invalid day!");

}

File: JAVA Fundamentals/Java Fundamentals/Arrays - Lab/01.DayOfWeek/P01_DayOfWeek.java
import java.util.Scanner;

public class P01_DayOfWeek {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Array
        String[] days = { "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday" };

        // Variable
        int day = Integer.parseInt(scanner.nextLine());

        // Conditional Statements
        if (day >= 1 && day <= 7)
            System.out.println(days[day - 1]);
        else
            System.out.println("Invalid day!");



    }
}
